cypress/e2e/apps-index.cy.js

```diff
@@ -0,0 +1,145 @@
+import { getDashboardConfig } from '../utils/getDashboardConfig';
+
+describe('Apps index', () => {
+  it('Redirects to app lists from root path if there are 2 or more apps', () => {
+    cy.intercept(
+      {
+        method: 'GET',
+        url: '/parse-dashboard-config.json',
+      },
+      getDashboardConfig([
+        {
+          appName: 'test1',
+        },
+        {
+          appName: 'test2',
+        },
+      ]),
+    );
+
+    cy.visit('/')
+    cy.wait(3000)
```

Comment:
Is it really necessary to wait here? Doesn't cypress handle the loading and continue once the page loading finished?